### PR TITLE
 implementation-auth-flow

### PR DESCRIPTION
**Pull request by Alanamiael**

### CODE REVIEW

File: src/components/Categories/helpers.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import business from '@assets/categories/business.svg';
+import general from '@assets/categories/general.svg';
+import entertainment from '@assets/categories/entertainment.svg';
+import health from '@assets/categories/health.svg';
+import science from '@assets/categories/science.svg';
+import technology from '@assets/categories/technology.svg';
+import sports from '@assets/categories/sports.svg';
```
Comment (Yarolaw) [inline src/components/Categories/helpers.ts]: use PNG here instead

File: src/components/Categories/index.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  const handleImageClick = (index: number) => {
+    const newActiveStates = [...activeStates];
+    newActiveStates[index] = !newActiveStates[index];
+    setActiveStates(newActiveStates);
+  };
+
+  return (
+    <div className={cl.container}>
+      {categoriesData.map(({ img, name }, index) => (
+        <div className={cl.category} key={index}>
```
Comment (Yarolaw) [inline src/components/Categories/index.tsx]: For `key` better use `name`

File: src/components/Categories/helpers.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+export const categoriesData = [
+  { img: general, name: 'General' },
```
Comment (Yarolaw) [inline src/components/Categories/helpers.ts]: please add the new key `active: false`

File: src/components/PhotoAdd/PhotoAdd.module.scss [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    margin: 28px 0 32px 0;
+  }
+  .buttonsContainer {
+    width: 100%;
+    text-align: center;
+    margin: 250px 0 12px 0;
+  }
+  .noUnderline {
+    text-decoration: none;
+    .skipTitle {
```
Comment (Yarolaw) [inline src/components/PhotoAdd/PhotoAdd.module.scss:31]: please add space between classnames

File: src/components/PhotoAdd/index.tsx [hunk 5]
```diff
@@ -0,0 +1,36 @@
+import { Link, useNavigate } from 'react-router-dom';
+import { ButtonTitles, Titles } from '@enums/enums';
+import CustomButton from '@shared/Button';
+import { MainTitle, RegularText } from '@shared/Typography';
+import defaultAvatar from '@assets/avatars/avatar.svg';
+import addAvatar from '@assets/avatars/addAvatar.svg';
+import cl from './PhotoAdd.module.scss';
+
+function PhotoAdd() {
```
Comment (Yarolaw) [inline src/components/PhotoAdd/index.tsx]: rewrite to arrow function

File: src/shared/Typography/index.tsx [hunk 6]
```diff
@@ -21,17 +21,25 @@ export const RegularText = styled('p')<RegularTextProps>(({ bold }) => ({
...
-    color === 'black'
-      ? 'var(--black-primary-color)'
-      : color === 'gray'
-      ? 'var(--black-lighter-color)'
-      : color === 'lightGray'
-      ? 'var( --grey-primary-color)'
-      : 'black',
-}));
+export const RegularBold = styled('p')<SubTitleProps>(({ color }) => {
+  const getColor = () => {
```
Comment (Yarolaw) [inline src/shared/Typography/index.tsx]: move `getColor` from `RegularBold`

File: src/ts/interfaces.ts [hunk 7]
```diff
@@ -39,3 +56,23 @@ export interface RegularTextProps {
 export interface FirstToKnowProps {
   onButtonClick: () => void;
 }
+export interface FormValuesProps {
```
Comment (Yarolaw) [inline src/ts/interfaces.ts]: `FormValuesProps` and `InitialValuesType` have 3 equal keys, we can optimize it with `Omit` ts utils (also please check the `Pick` and use it in the future)

Comment (Alanamiael) [reply src/ts/interfaces.ts]: FormValuesProps interface is not used anywhere, so it can be deleted.

**Pull request merged**